backend/app.py

```diff
@@ -165,6 +165,19 @@ def update_account(user):
     return {}, 204
 
 
+@app.route('/portal/logout', methods=['GET'])
+@token_required
+def logout(user):
+    """Logs the user out by returning an invalid JWT"""
+    data = {}
+    token = jwt.encode(data, 'LOGGEDOUT', algorithm="HS256")
+
+    resp = make_response(jsonify({'msg': "Erfolgreich ausgeloggt."}), 200)
+    resp.set_cookie('JWT', token, samesite=None)
```

Comment:
Würde es nicht reichen den Cookie zu löschen anstatt einen neuen invalid token zu setzen

Reply:
Es gibt anscheinend keine Möglichkeit den Cookie zu löschen. Die gängigste Methode soll sein, den Cookie mit einem Dummy Wert zu versehen und ihn mit einem Verfallsdatum in der Vergangenheit zu versehen.
[Flask: How to remove cookies?](https://stackoverflow.com/questions/14386304/flask-how-to-remove-cookies)

Reply:
Ich füge noch eine kleine Information dem Cookie hinzu. `"exp": 0`, dann ist der Token definitiv als abgelaufen versehen.